Xwt.XamMac/Xwt.Mac/OutlineViewBackend.cs

```diff
@@ -26,6 +26,8 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
+global using ObjCRuntime;
```

Comment:
In every other repo I've been actually adding `using ObjCRuntime;` in each file that needed it. Another option would be to allow implicit usings and specify it there. I have no preference. I did the `global using` shortcut here to save some time.

Reply:
I am okay with global using here or at MSBuild / AssemblyInfo.cs level.